### ISSUE TITLE
Sorce of the drivers?

### ISSUE THREAD

**Issue by ghost** [Elif-dot/RTL8761BU#1]
Where/how do you find these drivers? Do they come from the manufacturer or a 3rd party? How did you figure out where to look to get them?

I have looked everywhere I can think of and found several unofficial downloads like this one that all have the same file structure, but I can't find the source of where they are coming from.

Thanks for the help even though this isn't relay an issue.

**Comment by Elif-dot** [comment]
They come from ASUS' official downloads.
The only thing I have modified is the ReadMe file, as the default instructions don't properly work on some GNU/Linux distributions.
https://www.asus.com/us/Networking/USB-BT500/HelpDesk_download/
I mainly had to use these drivers as the ones in the mainstream kernel are for a completely different bluetooth chipset and don't work with the BT500. I had a headache while instaling them as the default instructions don't properly work with Fedora, so I had to look up a bunch of solutions on different forums.
I eventually just modified the ReadMe and reposted the drivers with the GPL 2.0 license so anyone can have them ready to download with the correct instructions.
